scipy/stats/morestats.py

```diff
@@ -2721,50 +2722,109 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False):
         The second set of measurements.  If `y` is not given, then the `x`
         array is considered to be the differences between the two sets of
         measurements.
-    zero_method : string, {"pratt", "wilcox", "zsplit"}, optional
+    zero_method : {"pratt", "wilcox", "zsplit"}, optional
         "pratt":
-            Pratt treatment: includes zero-differences in the ranking process
-            (more conservative)
+            includes zero-differences in the ranking process,
+            but drop ranks of the zeros, see [4]_, (more conservative)
         "wilcox":
-            Wilcox treatment: discards all zero-differences
+            Wilcox treatment: discards all zero-differences, the default
```

Comment:
I think putting the default explanation just under the `zero_method` is better for visibility/readability just like you did in `alternative` docstring.

Reply:
1. `Pratt treatment: ` was dropped, but `Wilcox treatment: ` was kept. What's the reasoning behind keeping one but not the other?

2. Is there a general standard for where to describe the "default" value? First? Last?

Reply:
1. I removed the names now for all options: they are not standard names and the description is enough in my view.

2. i kept the alphabetical order, happy to change it.